test/unit/presenters/formats/completed_transaction_presenter_test.rb

```diff
@@ -75,7 +75,6 @@ def result
     edition.update_attribute(:slug, 'foo')
     expected = [
       { path: '/foo', type: 'prefix' },
```

Comment:
I don't believe so, as far as I understand ruby, trailing commas are optional. I think the GDS styleguide even encourages it https://github.com/alphagov/styleguides/blob/master/ruby.md